posthog/settings.py

```diff
@@ -371,6 +371,10 @@ def print_warning(warning_lines: Sequence[str]):
 CELERY_QUEUES = (Queue("celery", Exchange("celery"), "celery"),)
 CELERY_DEFAULT_QUEUE = "celery"
 CELERY_IMPORTS = ["posthog.tasks.webhooks"]  # required to avoid circular import
+
+if PRIMARY_DB == CLICKHOUSE:
```

Comment:
Or we could just to a try import to test if it exists and then append?

Reply:
Yeah we already do that on line 234 and 499